pennylane/math/quantum.py

```diff
@@ -455,6 +455,71 @@ def reduced_dm(state, indices, check_state=False, c_dtype="complex128"):
     return density_matrix
 
 
+def purity(state, indices, check_state=False, c_dtype="complex128"):
+    r"""Computes the purity from a state vector or density matrix.
+
+    .. math::
+        \gamma = Tr(\rho^2)

```

Comment:
Just to keep the same style as `vn_entropy` and `relative_entropy`.
```suggestion
        \gamma = \text{Tr}(\rho^2)
```